benchmark/src/main/java/com/linecorp/decaton/benchmark/RecordsGenerator.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.decaton.benchmark;
+
+import java.util.ArrayDeque;
+import java.util.Deque;
+import java.util.Properties;
+import java.util.Random;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.apache.kafka.common.serialization.ByteArraySerializer;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class RecordsGenerator {
+    public static final int LATENCY_CYCLE = 1000;
+
+    private final int[] latencies;
+
+    public RecordsGenerator(int maxProcessLatencyMs) {
+        latencies = new int[LATENCY_CYCLE];
+        // Use maxProcessingLatencyMs as the seed so that we always get consistent
+        // set of latencies.
+        Random random = new Random(maxProcessLatencyMs);
+        for (int i = 0; i < latencies.length; i++) {
+            latencies[i] = random.nextInt(maxProcessLatencyMs);
```

Comment:
I'm not sure adding random latencies is meaningful.
It can make benchmark result unstable and, to observe performance it's enough to just add constant latency I think?

Reply:
As in `maxProcessLatencyMs` is used as random seed, actually the generated sequence is always the same and stable for every benchmark.
What I wanted here is uniformly distributed latency between 0..max with O(1) space but always stay same for different benchmark where latency bound is the same.

Reply:
I see.
But my point is, adding random latency itself (regardless it generates same sequence every time) is meaningful for benchmarking.

Reply:
I considered about the point of adding variations for simulated latency and turns out that it might be useful only when we wanna test decaton's isolation functionality that alleviates impact of slowly processed task X from other tasks in the same partition if they have different keys. However, to test that functionality particularly it is best to do in unit test confirming that they are processed by different threads independently rather than benchmark.
So now consistently simulated latency is better as in it provides much predictable result. 
Let me switch like that then.